openbas-api/src/main/java/io/openbas/utils/pagination/SortUtilsRuntime.java

```diff
@@ -0,0 +1,124 @@
+package io.openbas.utils.pagination;
+
+import io.openbas.helper.SupportedLanguage;
+import io.openbas.utils.schema.PropertySchema;
+import io.openbas.utils.schema.SchemaUtils;
+import org.springframework.data.domain.Sort;
+
+import javax.annotation.Nullable;
+import java.lang.reflect.Field;
+import java.util.*;
+import java.util.Map.Entry;
+
+import static io.openbas.config.SessionHelper.currentUser;
+import static io.openbas.utils.schema.SchemaUtils.*;
+import static java.util.Comparator.comparing;
+import static org.springframework.util.StringUtils.hasText;
+
+public class SortUtilsRuntime {
+
+  public static Sort toSortRuntime(@Nullable final List<SortField> sorts) {
+    List<Sort.Order> orders;
+
+    if (sorts == null || sorts.isEmpty()) {
+      orders = List.of();
```

Comment:
What is the default value if sorting is not defined? By what property are the results sorted?

Reply:
If it's not defined, there are no sorting

Reply:
ok, sorted as ddbb result (maybe by PK)